examples/plotting/file/slider_callback_policy.py

```diff
@@ -20,10 +20,10 @@
       };
     }
 
-    para.set('text', "Slider Values\\n\\n  Slider 1: {0}\\n  Slider 2: {1}\\n  Slider 3: {2}".format(s1, s2, s3))
+    para.set('text', "<H1>Slider Values</H1><P>Slider 1: {0}<P>Slider 2: {1}<P>Slider 3: {2}".format(s1, s2, s3))
 """)
 
-para = PreText(text = "Slider Values:\n\n  Slider 1: 0\n  Slider 2: 0\n  Slider 3: 0", width = 200, height = 150)
+para = Div(text = "<H1>Slider Values:</H1><P>Slider 1: 0<P>Slider 2: 0<P>Slider 3: 0", width = 200, height = 150, render_as_text = False)
```

Comment:
Corrected.  @mattpap has hit me up on this as well.  Just a habit from my personal style.